test/Sql/Platform/Sqlite/SelectDecoratorTest.php

```diff
@@ -0,0 +1,87 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2016 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendTest\Db\Sql\Platform\Sqlite;
+
+use Zend\Db\Sql\Platform\Sqlite\SelectDecorator;
+use Zend\Db\Sql\Select;
+use Zend\Db\Adapter\ParameterContainer;
+use Zend\Db\Adapter\Platform\Sqlite as SqlitePlatform;
+
+class SelectDecoratorTest extends \PHPUnit_Framework_TestCase
+{
+    /**
+     * @testdox integration test: Testing SelectDecorator will use Select an internal state to prepare a proper limit/offset sql statement
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::prepareStatement
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::processLimit
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::processOffset
+     * @dataProvider dataProvider
+     */
+    public function testPrepareStatement(Select $select, $expectedSql, $expectedParams)
+    {
+        $driver = $this->getMock('Zend\Db\Adapter\Driver\DriverInterface');
+        $driver->expects($this->any())->method('formatParameterName')->will($this->returnValue('?'));
+
+        // test
+        $adapter = $this->getMock(
+            'Zend\Db\Adapter\Adapter',
+            null,
+            [
+                $driver,
+                new SqlitePlatform()
+            ]
+        );
+
+        $parameterContainer = new ParameterContainer;
+        $statement = $this->getMock('Zend\Db\Adapter\Driver\StatementInterface');
+        $statement->expects($this->any())->method('getParameterContainer')->will($this->returnValue($parameterContainer));
+
+        $statement->expects($this->once())->method('setSql')->with($expectedSql);
+
+        $selectDecorator = new SelectDecorator;
+        $selectDecorator->setSubject($select);
+        $selectDecorator->prepareStatement($adapter, $statement);
+
+        $this->assertEquals($expectedParams, $parameterContainer->getNamedArray());
+    }
+
+    /**
+     * @testdox integration test: Testing SelectDecorator will use Select an internal state to prepare a proper limit/offset sql statement
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::getSqlString
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::processLimit
+     * @covers Zend\Db\Sql\Platform\Sqlite\SelectDecorator::processOffset
+     * @dataProvider dataProvider
+     */
+    public function testGetSqlString(Select $select, $ignore, $alsoIgnore, $expectedSql)
```

Comment:
Can you give a more descriptive name, please? What is the behavior you're testing? E.g., "testGetSqlStringCorrectlyGeneratesUnionSyntaxFromCombine".